### PR TITLE
DM members who receive a ⚠️ warning from moderators

### LINKED ISSUE #78: DM members who receive a ⚠️ warning from moderators
The ⚠️ reaction can only be triggered by moderators. Currently it updates #mod-log with a copy of the message, but it should also DM the member who sent the message. I'm imagining something like

> You've received a warning from the moderators on your message in #channel: 
> ```
> message content
> ```
> link: https://discord.com/channels/xxx/yyy/zzz

In the future, maybe we could expand this to have a "discussion channel" for the user to privately discuss the situation with all mods.

### PR DESCRIPTION
**Pull request by Huuums**
- fix #78 
- Added a very handholdy contributing.md to set up the bot

### CODE REVIEW

**Comment by Huuums** [comment]
after checking out the other issues I also added a fix for #68

File: src/features/emojiMod.ts [hunk 1]
```diff
@@ -97,6 +98,17 @@ const reactionHandlers: ReactionHandlers = {
...
     }
+
+    const privateMessageToSender = [
+      `You've received a warning from the moderators on your message in ${message.channel}`,
+      "\n\n",
+      "Your message: \n",
+      `\`${truncateMessage(message.content)}\``,
+      "\n\n",
+      `Link: https://discord.com/channels/${message.guild?.id}/${message.channel.id}/${message.id}`
+    ].join("");
```
Comment (vcarl) [inline src/features/emojiMod.ts:109]: Just curious, why not a multiline string? This seems much noisier
```suggestion
    const privateMessageToSender = 
      `You've received a warning from the moderators on your message in ${message.channel}

Your message:
\`${truncateMessage(message.content)}\`

Link: https://discord.com/channels/${message.guild?.id}/${message.channel.id}/${message.id}
```

Comment (Huuums) [reply src/features/emojiMod.ts:109]: There's no specific reason. To be honest I just didn't think about it.

https://github.com/reactiflux/reactibot/blob/546418eea4b7d99a2cc4b324cd141a703a13142e/src/features/emojiMod.ts#L67

I copied this message and modified the text a bit ignoring the fact that there's some filtering going on in the example.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Contributing
```
Comment (vcarl) [inline CONTRIBUTING.md:1]: :100: this is super awesome to have, thanks for adding it.

File: src/utils.ts [hunk 3]
```diff
@@ -9,3 +9,15 @@ export const isStaff = (member: GuildMember | null | undefined) => {
     staffRoles.includes(role.name.toLowerCase())
   );
 };
+
+//Discords limit for messagelength
```
Comment (vcarl) [inline src/utils.ts:13]: just gonna
```suggestion
// Discord's limit for message length
```
real quick

**Review by vcarl — APPROVED** [review]
I think I'd like to adjust the format of the message we send (a ` within the message I think will bork the formatting as-is) but that's not a blocker for merging. Thanks for doing this!!

**Pull request merged**